packages/seacas/libraries/ioss/src/pamgen/Iopg_DatabaseIO.h

```diff
@@ -101,8 +101,8 @@ namespace Iopg {
     std::string title() const { return databaseTitle; }
     int         maximum_symbol_length() const override { return 32; }
 
-    void compute_block_membership(Ioss::SideBlock *         efblock,
-                                  std::vector<std::string> &block_membership) const;
+    void compute_block_membership__(Ioss::SideBlock *         efblock,
```

Comment:
If `block_membership` is empty on entrance and you mean to create it, it's perfectly efficient just to return the `std::vector` by value.  Return value optimization will avoid reallocation and copying.

Reply:
It is not necessarily empty on entry.  Same vector will be passed in for each block in the model and additional names pushed back to end.  With new move semantics, it is probably ok to return by value now, but originally was written pre C++11.  Will look into modernizing...

Reply:
Thanks!